File: fibonacci.py
# def getting():
#     x = int(input("enter a range : "))


#     for i in range(x):
#         if i == 0 or i == 1:
#             print(i)
#         else:
#             print()
    

     
# def fibonacci(n):
#     fib_sequence = [0, 1]
#     for i in range(2, n):
#         next_number = fib_sequence[-1] + fib_sequence[-2]
#         fib_sequence.append(next_number)
#     return fib_sequence

# # Example usage:
# n = 10  # Change n to the number of Fibonacci numbers you want
# print(fibonacci(n))



def fib_series(n):
    fib_list = [0, 1]
    for i in range(2,n):
        next_number = fib_list[-1]+fib_list[-2]
        fib_list.append(next_number)
    for num in fib_list:
        print(num)

n = 10
fib_series(10)

File: master/instant delete/fibonacci.cpp
// #include <iostream>
// using namespace std ;

// int main(){
//     int num;
//     int next_digit;
//     cout << "enter a number for print fibonacci series : " ;
//     cin >> num ; 
//     // num = 10;

//     int a = 0;
//     int b = 1 ;
//     cout << a << endl << b << endl ;

//     for (int i = 0 ; i < num-2 ; i++  ){
//         next_digit = a+b;
//         cout << next_digit << endl;
//         a = b ;
//         b = next_digit;
//     }
//     return 0;
// }



// fibonacci series using recursion

#include <iostream>
using namespace std ;

int fibonacci(int num){
    if (num == 0){
        return 0;
    }
    else if( num == 1){
        return 1;
    }
    else{
        return fibonacci(num-1) + fibonacci(num - 2); 
    }
}


int main(){
    int num;
    cout << "enter a number for print fibonacci series : " ;
    cin >> num ; 
    // num = 10;
    for (int i = 0 ; i < num ; i++ ){
    cout << fibonacci(i) << endl ;
    }


    return 0;
}